exercises/hamming/cases_test.go

```diff
@@ -64,4 +64,10 @@ var testCases = []struct {
 		0,
 		true,
 	},
+	{ // check for runes
+		"èà",
+		"èò",
```

Comment:
The cases_test.go is a generated file from the hamming [test case generator](https://github.com/exercism/go/blob/master/exercises/hamming/.meta/gen.go) whose input is the [problem specifications canonical-data.json](https://github.com/exercism/problem-specifications/blob/master/exercises/hamming/canonical-data.json).  Because of the generated cases_test.go, I would normally suggest doing a pull request against problem-specifications repo to add the test case in the canonical-data.json.

However, since this test case addition is UTF-8, and not all language tracks can accept UTF-8  input. So, if this test case is to be added, the better approach is to put the new test case inside hamming_test.go; the extra test case can be defined with same struct signature which matches cases_test.go testCases. The for loops in hamming_test.go  can use an append(testCases, extraTestCases...) to easily incorporate the Go specific test case(s).